src/main/java/frc/robot/Controls.java

```diff
@@ -272,6 +324,14 @@ private void configureElevatorBindings() {
                         .withTimeout(0.3))
                 .ignoringDisable(true)
                 .withName("Reset elevator zero"));
+    elevatorTestController
+        .leftBumper()
+        .whileTrue(
+            Commands.startEnd(
+                    () -> s.elevatorSubsystem.setCoastMode(),
+                    () -> s.elevatorSubsystem.setBrakeMode())
+                .ignoringDisable(true)
+                .withName("Coast to Brake Mode"));
```

Comment:
```suggestion
    elevatorTestController.leftBumper().whileTrue(s.elevatorSubsystem.holdCoastMode());
```
This goes hand in hand with the changes to the elevator factories. (Also, I double checked and this actually is a free button on elevatorTestController)